projects/sandbox/train/train.py

```diff
@@ -75,14 +75,17 @@ def main(
     # and validation if valid_frac specified
     background = load_background(background_path, ifos)
 
+    # intrinsic parameters is an array of shape (n_params, n_signals)
     signals, intrinsic = load_signals(waveform_dataset, inference_params)
 
     if valid_frac is not None:
         background, valid_background = split(background, 1 - valid_frac, 1)
```

Comment:
In line with the previous comment above, let's also add a comment here about the shape of the background

Reply:
sounds good